docs/getting-started.md

```diff
@@ -142,35 +85,44 @@ const getTodos = `
 `;
 
 const TodoList = ({ limit = 10 }) => {
-  const [res] = useQuery({
+  const [{ fetching, data, error, extensions }] = useQuery({
```

Comment:
Let’s keep destructuring separately here, and name `res` as `queryResult` to convey its purpose here